### PR TITLE
Updating comments for splitWhenever.js

### PR DESCRIPTION
**Pull request by mudit56**
updating comments to make the function correctly documented.

### CODE REVIEW

File: source/splitWhenever.js [hunk 1]
```diff
@@ -6,10 +6,9 @@ import _curryN from './internal/_curryN';
  * @memberOf R
  * @since v0.26.1
  * @category List
- * @sig (a -> Boolean) -> [a] -> [[a]]
- * @param {Number} n
- * @param {Array} list
- * @param {Boolean} incDelimiter
+ * @sig (a -> Boolean) -> [a] -> []
```
Comment (Bradcomp) [inline source/splitWhenever.js]: Thanks for the update to the params, however I think the @sig was correct initially. Can you change it back?

Comment (mudit56) [reply source/splitWhenever.js]: Sure.

Comment (mudit56) [reply source/splitWhenever.js]: Done

**Comment by CrossEye** [comment]
Thank you, @mudit56!

**Pull request merged**